util/cli_ui.go

```diff
@@ -39,12 +39,18 @@ type ColorizeUi struct {
 	OutputFormat *output.OutputFormat
 }
 
-var UI *ColorizeUi
+var UI = ColorizeUi{
+	Colorize: Colorize(),
+	Ui: &cli.BasicUi{
+		Writer:      os.Stdout,
+		ErrorWriter: os.Stderr,
+	},
+}
```

Comment:
This change was made so we don't get a panic for de-referencing a nil pointer if a project has not called InitUI for message like:
```
util.UI.Error("Could not initialize http client, failing.")
```